app/src/main/java/org/mozilla/fenix/settings/wallpaper/Extensions.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.wallpapers
+
+import java.util.Locale
+
+/**
+ * The extension function to group wallpapers according to their name.
+ **/
+
+fun List<Wallpaper>.groupByCollection(): Map<Wallpaper.Collection, List<Wallpaper>> = groupBy {
+    it.collection
+}.filter {
+    it.key.name != "default"
+}.map {
+    if (it.key.name == "classic-firefox") {
+        it.key to listOf(Wallpaper.Default) + it.value
+    } else {
+        it.key to it.value
+    }
+}.toMap()
+
+/**
+ * The extension function for formatting wallpaper collection title.
+ **/
+
+fun Wallpaper.Collection.getDisplayName(): String = this.name
+    .split('-')
+    .joinToString(" ") { word ->
+        word.replaceFirstChar {
+            if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString()
+        }
+    }
```

Comment:
We can remove this since we will just use a string resource for "Classic Firefox"